### PR TITLE
Status Widget

### PR DESCRIPTION
**Pull request by swissmanu**
please find a new status widget in this pull request :)

### CODE REVIEW

**Comment by swissmanu** [comment]
any idea why the unit test fails on travis? it runs flawlessly on my machine 😕

File: src/components/Status.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        });
+    },
+
+    render() {
+        var widgetBodyNode = (<div className="widget__body" />);
+
+        if (this.state.status) {
+            var messageNode = null;
+
+            if(this.state.status.status !== 'good') {
```
Comment (plouc) [inline src/components/Status.jsx]: missing space `if (`

File: src/components/StatusIcon.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+var React            = require('react');
```
Comment (plouc) [inline src/components/StatusIcon.jsx]: extra white spaces

File: src/components/StatusIcon.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    propTypes: {
+        status: React.PropTypes.string.isRequired,
+        message: React.PropTypes.string.isRequired
+    },
+
+    render() {
+        var iconClasses = 'fa fa-';
+        var statusClasses = 'github__status__current__icon github__status__current__icon--';
+
+        if(this.props.status.toLowerCase() === 'good') {
```
Comment (plouc) [inline src/components/StatusIcon.jsx]: missing space `if (`

File: src/components/index.js [hunk 4]
```diff
@@ -3,5 +3,6 @@ module.exports = {
     PullRequests:                require('./PullRequests.jsx'),
     IssueLabelsTreemap:          require('./IssueLabelsTreemap.jsx'),
     UserBadge:                   require('./UserBadge.jsx'),
-    RepositoryContributorsStats: require('./RepositoryContributorsStats.jsx')
+    RepositoryContributorsStats: require('./RepositoryContributorsStats.jsx'),
+	Status:                      require('./Status.jsx')
```
Comment (plouc) [inline src/components/index.js]: spaces

**Comment by swissmanu** [comment]
damn different editor settings ;) thanks for the review! gonna fix it up.

**Comment by swissmanu** [comment]
hey @plouc, did you find time to look into again?
is there anything i can add or are you fine with the PR? :)

**Comment by plouc** [comment]
@swissmanu sorry for the delay, didn't took the time to review it, thanks for your contribution !

**Comment by swissmanu** [comment]
dont worry at all... i was just curious :) cool it made it into the repo :+1:

**Pull request merged**